src/lib.rs

```diff
@@ -23,13 +23,9 @@ extern crate zmq;
 use std::mem::transmute;
 use time::precise_time_ns;
 
-use radix::PrefixTree;
```

Comment:
Looks like this might be the only usage of the `radix` crate? If so, can it be removed from `Cargo.toml` and the `extern crate radix` call above?